recipe/conda_build_config.yaml

```diff
@@ -49,8 +49,7 @@ fortran_compiler:              # [unix or win64]
   - flang                      # [win64]
 fortran_compiler_version:      # [unix or win64]
   - 2                          # [(linux64 or osx) and (environ.get('CF_COMPILER_STACK') == 'comp4')]
-  - 4                          # [osx]
```

Comment:
Shouldn't this be kept as it is?

Reply:
Do we want to build both? IDK if we answered that question.

Reply:
We did build both when I merged those test PRs, but I guess we could do this globally.

Reply:
OK. I kept this line so we build everything. When defaults changes we can remove at a later date.